### PR TITLE
fix: added check to check if package.json contains output

### PR DESCRIPTION
**Pull request by zZHorizonZz**
## Describe your changes

This PR adds a check to ensure that if `package.json` doesn't contain an `output` field, we don't get a null pointer exception, but instead, get an appropriate warning.

Error in question:
```
[error]: Build step io.quarkiverse.quinoa.deployment.QuinoaProcessor#prepareQuinoaDirectory threw an exception: java.lang.NullPointerException: Cannot invoke "jakarta.json.JsonString.getString()" because the return value of "org.eclipse.parsson.JsonObjectBuilderImpl$JsonObjectImpl.getJsonString(String)" is null
[ERROR] 	at org.eclipse.parsson.JsonObjectBuilderImpl$JsonObjectImpl.getString(JsonObjectBuilderImpl.java:249)
[ERROR] 	at io.quarkiverse.quinoa.deployment.framework.override.NextFramework.override(NextFramework.java:34)
[ERROR] 	at io.quarkiverse.quinoa.deployment.framework.FrameworkType.overrideConfig(FrameworkType.java:91)
[ERROR] 	at io.quarkiverse.quinoa.deployment.QuinoaProcessor.prepareQuinoaDirectory(QuinoaProcessor.java:109)
[ERROR] 	at java.base/java.lang.invoke.MethodHandle.invokeWithArguments(MethodHandle.java:733)
[ERROR] 	at io.quarkus.deployment.ExtensionLoader$3.execute(ExtensionLoader.java:856)
[ERROR] 	at io.quarkus.builder.BuildContext.run(BuildContext.java:256)
[ERROR] 	at org.jboss.threads.ContextHandler$1.runWith(ContextHandler.java:18)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
[ERROR] 	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
[ERROR] 	at java.base/java.lang.Thread.run(Thread.java:1570)
[ERROR] 	at org.jboss.threads.JBossThread.run(JBossThread.java:483)
```
 

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.

### CODE REVIEW

File: deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java [hunk 1]
```diff
@@ -14,6 +14,7 @@
 public class NextFramework extends GenericFramework {
 
     private static final Logger LOG = Logger.getLogger(NextFramework.class);
+    private static final String OUTPUT = "export";
```
Comment (ia3andy) [inline deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java:17]: where is this used?

File: deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java [hunk 2]
```diff
@@ -31,8 +32,8 @@ public QuinoaConfig override(QuinoaConfig delegate, Optional<JsonObject> package
                     LOG.warn(
                             "Make sure you define  \"build\": \"next build \", in the package.json to make Next work with Quinoa.");
                 }
-                String output = packageJson.get().getString("output");
-                if (!"export".equals(output)) {
+                
+                if (!packageJson.get().containsKey("output") || !OUTPUT.equals(packageJson.get().getString("output"))) {
```
Comment (ia3andy) [inline deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java:36]: AFAIK `packageJson.get().getString("output")` will return null if it's not defined and the check will be false, so what's the difference?

Comment (ia3andy) [reply deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java:36]: Ahh we are using Jakarta JsonObject not Vertx, ok implementation are different it seems, you need to use `getString("output", null)`

File: deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java (hunk 1 above)
Comment (zZHorizonZz) [reply deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java:17]: This is just a small thing. I can remove it if it's not consistent with the code style. I just personally think that in the future, if the code for this gets bigger and there needs to be a change to this, you can change it at the top of the file and won't need to search for it.

File: deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java (hunk 2 above)
Comment (zZHorizonZz) [reply deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java]: Ok, I didn't know you can do that either.

File: deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java (hunk 1 above)
Comment (ia3andy) [reply deployment/src/main/java/io/quarkiverse/quinoa/deployment/framework/override/NextFramework.java:17]: all good could you call it `EXPECTED_OUTPUT_VALUE`

**Review by ia3andy — APPROVED** [review]
Thanks!!!

**Review by melloware — APPROVED** [review]

**Comment by melloware** [comment]
@all-contributors add @zZHorizonZz for code,bug

**Pull request merged**